plugins/source/aws/resources/services/route53/domains.go

```diff
@@ -111,14 +116,9 @@ func Domains() *schema.Table {
 			},
 			{
 				Name:     "status_list",
-				Type:     schema.TypeJSON,
+				Type:     schema.TypeStringArray,
```

Comment:
@hermanschaaf - A StringArray is a subset of JSON, maybe we can override the codegen to keep it as a JSON?

Reply:
Indeed we can... although it is a different question from whether we should 🙂 Future cases will also inevitably arise where we don't have a backwards-compatible option. Plus, just because we keep the column JSON, doesn't mean users don't need to update their queries (since the contents will change). I'd rather indicate that with a major version bump. 

Maybe we should do updates like these on a semi-regular schedule, but not so regularly so as to cause constant toil for users. Will need to think about this some more...

Reply:
This change only occurred because the codegen was using [v1](https://github.com/aws/aws-sdk-go-v2/blob/9c92421d2b2253960dd282b5966f7712b62469dd/service/route53domains/api_op_GetDomainDetail.go#L135) of the AWS Go SDK  which relied heavily on pointers rather than native go types like the [AWS SDK Go V2](https://github.com/aws/aws-sdk-go-v2/blob/9c92421d2b2253960dd282b5966f7712b62469dd/service/route53domains/api_op_GetDomainDetail.go#L135) does

Reply:
Ah, I see! So it shouldn't happen too often then. That's good to know

Reply:
AWS is really good about not releasing breaking changes, so unless something in our codegen/sdk changes as then we should always be good regenerating the clients